marlowe-playground-client/package.json

```diff
@@ -17,15 +17,14 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.10.2",
     "@popperjs/core": "^2.9.2",
-    "big-integer": "^1.6.50",
     "bignumber": "^1.1.0",
-    "blockly": "^4.20201217.0",
+    "blockly": "^9.3.3",
     "bootstrap": "^4.3.1",
     "decimal.js": "^10.0.0",
     "json-bigint": "^1.0.0",
-    "monaco-editor": "^0.22.3",
-    "monaco-emacs": "^0.2.2",
-    "monaco-vim": "^0.1.7",
+    "monaco-editor": "^0.38.0",
```

Comment:
Monaco editor didn't bring too much trouble that I could tell, just some codeAction changes